packages/core/lib/debug/vscode.js

```diff
@@ -0,0 +1,73 @@
+const childProcess = require("child_process");
+
+class VSCodeDebugger {
```

Comment:
Just a small detail, the name `VSCodeDebugger` might be a bit misleading. What about `VSCodeOpenDebugger` or `VSCodeDebugAdapter`?